test/test_boundary_conditions_integration.jl

```diff
@@ -448,6 +427,34 @@ test_boundary_conditions(C, FT, ArrayType) = (integer_bc(C, FT, ArrayType),
             @info "  Testing open boundary conditions [$A, $FT]..."
             test_flat_extrapolation_open_boundary_conditions(arch, FT)
             test_perturbation_advection_open_boundary_conditions(arch, FT)
+
+            # Only PerturbationAdvectionOpenBoundaryCondition
+            U₀ = 1.0
```

Comment:
```suggestion
            U₀ = 1
```

in case an impressionable user reads it